src/constants.js

```diff
@@ -0,0 +1,20 @@
+/**********************************************************************************************************************
+ * This module contains constants that are meant to be used throughout the entire application.  Each constant is
+ * defined using the define() function which ensures that it remains read-only when this module is consumed.
+ **********************************************************************************************************************/
+
+function define(name, value) {

```

Comment:
Do you have a C background? This is very C looking to me 😄 . Id prefer more es6 style constant declaration


`export const APPLICATION_NAME = 'Voice Desktop`;
etc

Reply:
My background is Java/C++/C#.  I've had to start learning JS/Node/Electron so that I can help contribute to this application (which ROCKS btw!).  So thanks for the suggestion here :)

Will doing what you propose create an actual constant though?  The `define()` technique I used above was something I discovered in a stackoverflow post.  I couldn't figure out how to make an actual constant in JS.  I.e., one that is actually unmodifiable.  JS seems to be anarchy where anyone can do anything.

What are your thoughts on this?  While the define() thing does create true constants, I also don't like how it prevents IntelliSense from working.

Reply:
If you really want the strings defined as constants you can do this:

`
const APPLICATION_NAME = 'Voice Desktop';
const APPLICATION_ICON_LARGE =  '1024px-Google_Voice_icon_(2020).png';

modules.exports = { APPLICATION_NAME, APPLICATION_ICON_LARGE
`